Lib/test/test_multibytecodec.py

```diff
@@ -168,16 +168,11 @@ def test_decode_unicode(self):
 class Test_StreamReader(unittest.TestCase):
     def test_bug1728403(self):
         try:
-            f = open(TESTFN, 'wb')
-            try:
-                f.write(b'\xa1')
-            finally:
-                f.close()
-            f = codecs.open(TESTFN, encoding='cp949')
-            try:
-                self.assertRaises(UnicodeDecodeError, f.read, 2)
-            finally:
-                f.close()
+            with open(TESTFN, 'wb') as fp:
+                fp.write(b'\xa1')
+
+            with open(TESTFN, encoding='cp949') as fp:
```

Comment:
bug1728403 originally was about `codecs.open()`. I don't know if there is a sense to use `io.open()` here.